client/state/account-recovery/actions.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+
+import {
+	ACCOUNT_RECOVERY_FETCH,
+	ACCOUNT_RECOVERY_FETCH_SUCCESS,
+	ACCOUNT_RECOVERY_FETCH_FAILED,
+} from 'state/action-types';
+
+export const accountRecoveryFetchSuccess = ( { email, email_validated, phone, phone_validated } ) => {
+	return {
```

Comment:
Why destructure? Why not just pass the entire response in the action object and let the reducer pick the parts it cares for?